.circleci/unittest/linux/scripts/run_test.sh

```diff
@@ -7,4 +7,7 @@ conda activate ./env
 
 export PYTORCH_TEST_WITH_SLOW='1'
 python -m torch.utils.collect_env
-pytest --cov=torchvision --junitxml=test-results/junit.xml -v --durations 20 test --ignore=test/test_datasets_download.py
+# test_onnx requires additional packages to be installed and has its own pipeline.
+pytest --cov=torchvision --junitxml=test-results/junit.xml -v --durations 20 test \
+  --ignore=test/test_datasets_download.py \
+  --ignore=test/test_onnx.py
```

Comment:
Let's remove this: I think it's actually valuable to see the tests being skipped in the CI: it shows they exist, and that they're skipped. If anything, it shows that the `skipif` logic actually works as expected.

As you said they're being run by another CI run anyway so there's no risk of missing them.

Reply:
Sorry, I forgot that part.

If you wish to remove the try/except, we should be able to convert it into `importorskip`: https://docs.pytest.org/en/latest/how-to/skipping.html#skipping-on-a-missing-import-dependency

Reply:
Nice. Done.